tdr-snapshot-examples/get_file_info_by_drs_path.py

```diff
@@ -0,0 +1,44 @@
+# Gets the report file information based off the provided drs path
+
+import json
+
+from obtain_token import obtain_session
+
+headers = {
+    "Accept": "application/json",
+    "User-Agent": "GPO Bound Query",
+    "Content-Type": "application/json",
+    "accept": "*/*",
+}
+
+
+def get_file_info_by_drs_path(session, drsPath):
```

Comment:
I think pass the header in as well.  it seems odd for it to just be floating

Reply:
Done by you 👍